underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -148,6 +152,9 @@ public HdfsUnderFileSystem(AlluxioURI ufsUri, UnderFileSystemConfiguration conf,
       // Set Hadoop UGI configuration to ensure UGI can be initialized by the shaded classes for
       // group service.
       UserGroupInformation.setConfiguration(hdfsConf);
+      Class.forName(HDFS_EC_CODEC_REGISTRY_CLASS);
```

Comment:
is it better to check `alluxio.UfsConstants.UFS_HADOOP_VERSION` here and only class load if the major version is 3.0 or later?

also this should be in its own separate try - catch block, which is best effort. 
I have seen `UserGroupInformation.setConfiguration(hdfsConf);` throw this `ClassNotFoundException` before

```java
try {
  ...
  try {
    Class.forName(HDFS_EC_CODEC_REGISTRY_CLASS);
  } catch (ClassNotFoundException e) {
    LOG.warn("Cannot initialize HDFS EC CodecRegistry. HDFS EC will not be supported.");
  }
}
```

Reply:
Hi Bin, thanks for your comment, I have made changes according to your suggestions. please review again.